asab/api/service.py

```diff
@@ -97,12 +99,12 @@ def initialize_zookeeper(self, zoocontainer=None):
 		# get zookeeper-serivice
 		self.ZkContainer = zoocontainer
 		self.ZkContainer.advertise(
-			data=self._build_zookeeper_adv_data(),
+			data=self._build_zookeeper_adv_data(adv_data = None),
 			path="/run/{}.".format(self.App.__class__.__name__),
 		)
 
 
-	def _build_zookeeper_adv_data(self):
+	def _build_zookeeper_adv_data(self ,adv_data):
```

Comment:
you don't need to pass `adv_data` into "build zookeeper adv data" method ... it is kind of "redundant" hence likely wrong.